src/components/sessions/CancelModal.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable react/jsx-sort-props */
```

Comment:
TODO @rschlaefli: I need to look at the lint rules in general, so we don't have to work around them like this.